shared/app.js

```diff
@@ -73,7 +74,9 @@ module.exports = Backbone.Model.extend({
      * Initialize the `ClientRouter` on the client-side.
      */
     if (!isServer) {
-      var ClientRouter = this.options.ClientRouter || require(defaultRouterModule);
```

Comment:
That's basically what i'm thinking for future, but don't want to make a breaking change like that in a minor version bump as we move towards that system.